src/contracts/atlas/GasAccounting.sol

```diff
@@ -228,7 +240,7 @@ abstract contract GasAccounting is SafetyLocks {
     /// @param result The result bitmap of the SolverOperation execution.
     /// @param bidFind Indicates if called in the context of `_getBidAmount` in Escrow.sol (true) or not (false).
     /// @param includeCalldata Whether to include calldata cost in the gas calculation.
-    function _releaseSolverLock(
+    function _handleSolverAccounting(
```

Comment:
This is only ever called with bidFind = false, so we can remove the bidFind param and simplify the logic a bit I think.

Reply:
https://github.com/FastLane-Labs/atlas/pull/271/commits/91b922594093e5e8e3f229877ab74831cc71b2ba

Reply:
good catch!